internal/error.go

```diff
@@ -645,6 +718,10 @@ func getErrType(err error) string {
 	for t = reflect.TypeOf(err); t.Kind() == reflect.Ptr; t = t.Elem() {
 	}
 
+	if t.Name() == goErrType {
+		return ""
+	}
```

Comment:
If it was standard error created with `errors.New` error type will be empty string. I think this is better for `Type` field.